setup.md

```diff
@@ -1,2 +1,4 @@
 ## Pre-requisites
-# To be added
\ No newline at end of file
+# Environment Variables
+
+- export userOrgServiceLogLevel=INFO
```

Comment:
userOrgServiceLogLevel this env need to be rename as "sunbird_user_org_log_level"